renku/cli/update.py

```diff
@@ -193,4 +194,12 @@ def update(client, revision, no_output, siblings, paths):
             )
         )
 
-    execute(client, output_file, output_paths=output_paths)
+    try:
+        execute(client, output_file, output_paths=output_paths)
+    except WorkflowRunError:
+        raise click.ClickException(
+            'Unable to finish re-executing workflow; check the workflow'
+            ' execution outline above and the generated {0} file for'
+            ' potential issues, then remove the {0} file and try again'.
+            format(output_file)
+        )
```

Comment:
Really minor issue, feel free to ignore it: You can also put this text in `WorkflowRunError` when raising it and do not catch it here. All `RenkuException` subclasses are treated gracefully as well.

Reply:
Thanks, I changed it to be more specific to this issue and be handled in this way.